lib/post_revisor.rb

```diff
@@ -95,6 +96,13 @@ def self.track_topic_field(field, &block)
     end
   end
 
+  track_topic_field(:featured_link) do |tc, featured_link|
```

Comment:
I would avoid using variable names like `tc` because it degrades the readability of the code.